src/SFA.DAS.LevyTransferMatching.Web/Models/Pledges/ApplicationApprovalOptionsViewModel.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Models.Pledges
+{
+    public class ApplicationApprovalOptionsViewModel : ApplicationApprovalOptionsPostRequest
+    {
+        public string EmployerAccountName { get; set; }
```

Comment:
We could retrieve this via a call into the outer api rather than adding it to the query string